Gopkg.toml

```diff
@@ -38,10 +38,6 @@
   name = "github.com/giantswarm/k8sclient"
   branch = "master"
 
-[[constraint]]
-  name = "github.com/giantswarm/k8shealthz"
```

Comment:
Super old, broken with microerror master, and just not needed. Dumped.

Reply:
FYI you should pin all giantswarm dependencies to `version = "~0.1.0"`. Then you should be fine.